### DISCUSSION TITLE
Any chance for implementing tilt support?

### DISCUSSION THREAD

**Discussion by Gregoras** [profi200/open_agb_firm#204] [Ideas]
I'm not knowing 3DS/NDS architecture very well, but
because OAF still have access to 3DS PPU (border support + screenshot feature), I guessing that with OAF loaded some low level 3DS features are still presented. I wonder if is possible to using 3DS built-in accelerometer and gyro to emulating cartridges with tilt sensor.

**Comment by profi200** [comment]
I theory it's possible. This requires 3 things:
* Per game patches.
* Drivers for the hardware on the 3DS side.
* A way to transfer data between 3DS and GBA side (breaking the "sandbox").

I currently only have the last one partially implemented.
